tests/testthat/test-rsc_filter.R

```diff
@@ -1,3 +1,9 @@
-test_that("multiplication works", {
-  expect_equal(2 * 2, 4)
-})
+  req <- make_req(
+    verb = "POST",
+    path = "/loess",
+    postBody = encode_payload(script = "concat", letters, .toJSON_args = NULL, raw = FALSE)
+  )
+
+test_that("rsc_filter() correctly rewrites requests", {
+
+})
```

Comment:
Still missing the test here

Reply:
Yeah, I just committed the test skeleton because I think this file's waiting on some changes, right? I'm happy to revert this change if you want, or — I should probably just add a `TODO` comment to make it clear it's intentional, eh?

Reply:
I added a "TODO" comment in the last commit.